packages/start-slicemachine/src/lib/setupSentry.ts

```diff
@@ -0,0 +1,70 @@
+import * as Sentry from "@sentry/node";
+
+import semver from "semver";
+import { App, H3Event, sendError } from "h3";
+import { SliceMachineManager } from "@slicemachine/manager";
+import { SENTRY_EXPRESS_DSN } from "../constants";
+
+export const setupSentry = async (
+	manager: SliceMachineManager,
+	app: App,
+): Promise<void> => {
+	!isDevEnv() || process.env.SENTRY_AUTH_TOKEN !== undefined;
+	if (!isSentryEnabled()) {
+		return;
+	}
+
+	const sliceMachineVersion =
+		await manager.versions.getRunningSliceMachineVersion();
+
+	let user;
+	try {
+		user = await manager.user.getProfile();
+	} catch {
+		// not logged in
+		// user stays undefined
+	}
+
+	const config = await manager.project.getSliceMachineConfig();
+
+	const isStableVersion =
+		/^\d+\.\d+\.\d+$/.test(sliceMachineVersion) &&
```

Comment:
Maybe this could use `semver.prerelease` instead to know if any prerelease is present